src/averageAges.js

```diff
@@ -38,6 +43,23 @@ function calculateMenAverageAge(people, century) {
  */
 function calculateWomenAverageAge(people, withChildren) {
   // write code here
+  const women = people.filter(el => el.sex === 'f');
+
+  let womenAverageAge = women.reduce((a, b) =>
+    a + (b.died - b.born), 0) / women.length;
+
+  womenAverageAge = +womenAverageAge.toFixed(2);
+
+  const womenWithChildren = women.filter((el, i, arr) =>
+    people.some(element => el.name === element.mother)
+  );
+
+  let womenAgeWithChildren = womenWithChildren.reduce((a, b) =>
+    a + (b.died - b.born), 0) / womenWithChildren.length;
+
+  womenAgeWithChildren = +womenAgeWithChildren.toFixed(2);
+
+  return withChildren === undefined ? womenAverageAge : womenAgeWithChildren;
```

Comment:
Here you can use short version, because `Boolean(undefined) === false`
```suggestion
  return withChildren ? womenAverageAge : womenAgeWithChildren;
```